TSSDistance.pm

```diff
@@ -72,13 +90,23 @@ sub run {
         $dist = $vf->start - $t->end;
     }
 
+    # Return upstream distance:
```

Comment:
And can be the if logics be changed to 
 ```
 if ($dist > 0) {
        return {
            TSSDistance => $dist,
        }
 }
    # Downstream distance only returned if both_direction flag is set to 1:
 if ($self->{both_direction} == 1){
        return {
            TSSDistance => $dist,
        }
 }
    # Otherwise return empty hash:


 return {};
```

Reply:
Sure! Good point.